src/app/utils/UserUtil.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * This file replicates and extends the contents of User.js in branches new-url-scheme && notifications-ui.
+ * Additions here need to be added to ./User.js when those are merged. Imports should pull that file, and this one should be deleted.
+ */
+
+
+let store = false;
+
+export const setStore = (theStore) => {
```

Comment:
It looks like this and the next 2 functions aren't used anywhere. Better to save them for a PR which introduce them?

Reply:
I want to get them into master so they _can be used_. I've seen 'cobbled together' url-gen going into master since this stuff was created. It's time to get ahead of the curve.

Reply:
Understood, but it makes for cleaner reviews. Otherwise, the next PR where you implement these functions it would be easy to assume the already-merged functions are tested where in fact they are untested dead code.

Reply:
They _are_ tested.